shared/styles/index.js.flow

```diff
@@ -29,6 +57,14 @@ declare export var windowStyle: {
   height: number,
 }
 
+declare export function platformStyles({|
+  common?: CSS._StylesCrossPlatform,
+  isIOS?: CSS._StylesMobile,
+  isAndroid?: CSS._StylesMobile,
+  isMobile?: CSS._StylesMobile,
+  isElectron?: CSS._StylesDesktop,
```

Comment:
why not `isDesktop`?

Reply:
because we have isElectron already in our constants/platform which is the source for isMobile etc